src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -924,14 +925,15 @@ public ResultSet getVersionColumns(String catalog, String schema, String table)
             throws SQLException {
         MongoJsonSchema botSchema =
                 createBottomSchema(
-                        new Pair<>(SCOPE, BSON_STRING.getBsonName()),
-                        new Pair<>(COLUMN_NAME, BSON_STRING.getBsonName()),
-                        new Pair<>(DATA_TYPE, BSON_INT.getBsonName()),
-                        new Pair<>(TYPE_NAME, BSON_STRING.getBsonName()),
-                        new Pair<>(COLUMN_SIZE, BSON_INT.getBsonName()),
-                        new Pair<>(BUFFER_LENGTH, BSON_INT.getBsonName()),
-                        new Pair<>(DECIMAL_DIGITS, BSON_INT.getBsonName()),
-                        new Pair<>(PSEUDO_COLUMN, BSON_INT.getBsonName()));
+                        new MongoJsonSchema.ScalarProperties(SCOPE, BSON_STRING, false),
```

Comment:
In this document, the SCOPE is a type `short` and doesn't state the column is nullable:
https://docs.oracle.com/javase/8/docs/api/java/sql/DatabaseMetaData.html#getVersionColumns-java.lang.String-java.lang.String-java.lang.String-